uni/lib/model/app_state.dart

```diff
@@ -40,6 +41,8 @@ class AppState {
       'feesStatus': RequestStatus.none,
       'lastUserInfoUpdateTime': null,
       'locationGroups': <LocationGroup>[],
+      'libraryOccupation': LibraryOccupation(0, 0),
```

Comment:
This can be null I think. The fetcher changes the variable

Reply:
When it is like this, the displayed occupation will be 0 until the real one is fetched. When set to null I'm getting an error displaying the widget.

Reply:
Can you add the refresh icon, instead of displaying 0?